dao-contracts/src/lib.rs

```diff
@@ -2,19 +2,19 @@
 
 #[doc(hidden)]
 pub mod action;
-mod admin;
-mod builder;
-mod dao_nft;
-mod kyc_voter;
+pub mod admin;
+pub mod builder;
+pub mod dao_nft;
+pub mod kyc_voter;
```

Comment:
Why make it public? A few lines below we reexport essential parts of modules.

Reply:
You are right.

Reply:
Agree.